### PR TITLE
Init Led Service files

### PR DESCRIPTION
**Pull request by Rjasuja**
Control LEDs on boot up to indicate the boot up cycle.

### CODE REVIEW

**Comment by avinash-palleti** [comment]
@lucasdemarchi we have verified this on Aero. LED is turning Green once boot process is completed.

File: recipes-support/notify-led/files/LICENSE [hunk 1]
```diff
@@ -0,0 +1,54 @@
+Apache License
```
Comment (lucasdemarchi) [inline recipes-support/notify-led/files/LICENSE]: Why are we adding licenses everywhere? meta layer already has a license that's applied to everything inside.

**Review by lucasdemarchi — CHANGES_REQUESTED** [review]

**Comment by Rjasuja** [comment]
Deleted License File and updated recipe file

**Comment by lucasdemarchi** [comment]
I'd prefer something that monitors the progress property as I mentioned before. But this is not critical for now so I'm merging it so we can move on.

**Pull request merged**